src/main/resources/templates/signup2.html

```diff
@@ -65,8 +70,20 @@ <h3 th:text="#{signup.createAcc.header}">Create Your Account</h3>
 			                            <li>&nbsp;</li>
 			                            <li>&nbsp;</li>
 			                        </ul>
-                                
-                                	<h3 th:text="#{signup.personalDetails.header}">Personal Details</h3>
+
+									<p class="account-change-fail" th:if="${#fields.hasErrors('firstName')}"><i class="fa fa-exclamation-circle"></i> <span th:errors="*{firstName}"></span></p>
+									<p class="account-change-fail" th:if="${#fields.hasErrors('lastName')}"><i class="fa fa-exclamation-circle"></i> <span th:errors="*{lastName}"></span></p>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)